setup.py

```diff
@@ -55,7 +55,7 @@
     license=about['__license__'],
     keywords='firebase cloud development',
     install_requires=install_requires,
-    packages=find_packages(exclude=['tests']),
+    packages=find_packages(include=['firebase*']),
```

Comment:
Thanks for pointing this out @fgmacedo. Can we drop the wildcard and explicitly make it `['firebase_admin']`?  We don't want anything else exposed from our binaries.